core/dovecot/conf/dovecot.conf

```diff
@@ -100,6 +100,7 @@ service auth-worker {
 protocol imap {
   mail_plugins = $mail_plugins imap_quota imap_sieve
   mail_max_userip_connections = 20
+  imap_idle_notify_interval = 29 mins
```

Comment:
I'm not sure if it is relevant. All documentation shows no space between the value and time unit. So 
```imap_idle_notify_interval = 29mins```
I do not know if this is significant, but let's follow the dovecot practices and use no space.
https://doc.dovecot.org/settings/core/ 
shows as default `2mins` (no space)
https://doc.dovecot.org/settings/types/#time
shows 1w, weeks and  7days. (no space)

Reply:
I've removed it